packages/angular_devkit/build_webpack/package.json

```diff
@@ -26,6 +26,6 @@
   },
   "peerDependencies": {
     "webpack": "^5.30.0",
-    "webpack-dev-server": "^5.0.2"
+    "webpack-dev-server": "^5.2.2"
```

Comment:
The peer dependency here has to stay the same, otherwise it will be a breaking change.

Reply:
Fixed!